src/main/groovy/com/readytalk/gradle/plugins/ci/CiLifecyclePlugin.groovy

```diff
@@ -0,0 +1,85 @@
+package com.readytalk.gradle.plugins.ci
+
+import com.readytalk.gradle.plugins.ci.tasks.CiTask
+import com.readytalk.gradle.util.PluginUtils
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.tasks.testing.Test
+import org.gradle.testing.jacoco.tasks.JacocoReport
+
+class CiLifecyclePlugin implements Plugin<Project>, PluginUtils {
```

Comment:
Again some refactoring here to make this class a bit easier to browse / understand.